modules/NewItemTracking.lua

```diff
@@ -210,6 +210,7 @@ end
 
 function mod:ShowBlizzardGlow(button, enable)
 	if enable then
+		local quality = select(4, GetContainerItemInfo(button.bag, button.slot))
```

Comment:
Let's do it without the use of the select function.

Reply:
It's a best coding practice to avoid unnecessary function calls. It's easily avoidable here.

Reply:
Well all other GetContainerItemInfo() calls in this addon use it.